sample-apps/generic_segmentation/main.py

```diff
@@ -59,6 +66,11 @@ def __init__(self, app_dir, studies):
     def init_infers(self):
         infers = {
             "segmentation": MyInfer([self.pretrained_model, self.final_model], self.network),
+            "Coldstart->ISeg+GraphCut": GenericISegGraphcutColdstart(),
+            "ISeg+GraphCut": GenericISegGraphCut(),
+            "ISeg+SimpleCRF": GenericISegSimpleCRF(),
+            # "ISeg+CRF": GenericISegCRF(), # disabled for now as MONAI CRF GPU has a bug
```

Comment:
lets remove these lines.. lets not confuse external users when we add them to main branch

Reply:
Sure, will remove it